allofplos/corpus_class.py

```diff
@@ -72,7 +77,7 @@ def dois(self):
     @property
     def filepaths(self):
         """List of article XML files in corpus directory, including the full path."""
-        return [os.path.join(self.directory, fname) for fname in self.files]
+        return [os.path.join(self.directory, fname) for fname in self.iter_files]
```

Comment:
well this could now be just `list(self.iter_filepaths)` like the other iter methods.

Reply:
fixed